src/screens/Menu/CommunitySettings.js

```diff
@@ -0,0 +1,159 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { Linking } from 'react-native';
+import ContainerWithHeader from 'components/Layout/ContainerWithHeader';
+import { BaseText } from 'components/Typography/Typography';
+import { ScrollWrapper } from 'components/Layout';
+import TextInput from 'components/TextInput';
+import Toast from 'components/Toast';
+import { spacing } from 'utils/variables';
+import { isValidEmail } from 'utils/validators';
+import { subscribeToNewsletter } from 'services/newsletter';
+import {
+  TWITTER_SOCIAL_ADDRESS,
+  TELEGRAM_SOCIAL_ADDRESS,
+  YOUTUBE_SOCIAL_ADDRESS,
+  MEDIUM_SOCIAL_ADDRESS,
+  FACEBOOK_SOCIAL_ADDRESS,
+  FORUM_SOCIAL_ADDRESS,
+} from 'constants/communityConstants';
+import { SettingsSection } from './SettingsSection';
+
+type Props = {
+
+};
+
+type State = {
+  email: string,
+};
+
+class CommunitySettings extends React.Component<Props, State> {
+  state = {
+    email: '',
+  }
+
+  goTo = (link: string) => () => Linking.openURL(link);
```

Comment:
Maybe it would be better to open an app if user has it on device rather than linking to webpage? Linking to webpage could be a fallback action.